app/app/Board.js

```diff
@@ -254,6 +275,15 @@ export class Board {
      */
     onMouseUp(event) {
```

Comment:
In my opinion you should select/unselect during mouse move not mouseUp.
Just a UX example select several files in Windows explorer. The files are being selected/unselectd on mousemove event. What do you think?